common-token/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![deny(warnings)]
+#![no_std]
+
+use soroban_sdk::{Address, Env};
+
+pub mod balance;
+pub mod storage;
+
+pub fn verify_caller_is_pool(e: &Env) -> Address {
+    let pool = crate::storage::read_pool(e);
+    pool.require_auth();
+    pool
+}
+
+pub fn check_nonnegative_amount(amount: i128) {
```

Comment:
I see that we use several prefixes (`require_`, `ensure_`, `check_`) for validation functions. Can we standardise the naming convention for such cases? I suggest the `require_` since it's a soroban naming for such cases.

Reply:
Let's discuss with team and establish convention